### DISCUSSION TITLE
Is there an explanation of each metric somewhere?

### DISCUSSION THREAD

**Discussion by mattt-seek** [change-metrics/monocle#1047] [General]
Most of them are self-explanatory, however I would love to know what "Iterations by change" and "Changes updates" mean

**Comment by morucci** [comment]
Hi,

There is an effort in progress to ensure that each metric are available via the catalog (https://demo.changemetrics.io/python/metrics). Each of them provides a Metric Description.

The "Change updates" from Lifecycle stats is based on "Change updates count" from the metric catalog and the description is "The metric is the count of commit push and force commit push events."

"Iteration by changes" is the ratio of "Change updates" and the count of changes created. https://github.com/change-metrics/monocle/blob/master/src/Monocle/Backend/Queries.hs#L946
